docs/maintain/maintain-guides-how-to-nominate-polkadot.md

```diff
@@ -57,25 +58,73 @@ detailed on the [account generation][] page.
+validators, if your active validator is oversubscribed, you will earn rewards only if your stake is
+within that of the top {{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominatorRewardedPerValidator" defaultValue={256}/> :polkadot }}{{ kusama: <RPC network="polkadot" path="consts.staking.maxNominatorRewardedPerValidator" defaultValue={256}/> :kusama }} nominators. If the validator misbehaves, It is worth noting that your
+stake is subject to slashing, irrespective of whether you are in the top {{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominatorRewardedPerValidator" defaultValue={256}/> :polkadot }}{{ kusama: <RPC network="polkadot" path="consts.staking.maxNominatorRewardedPerValidator" defaultValue={256}/> :kusama }} nominators or not.
+
+As the minimum active bond is a dynamic value, it can make your nomination inactive when the
+threshold goes above your bonded balance. Hence, to be eligible to earn rewards while nominating,
+you would need to stake a much higher balance than the minimum active bond.
+
+Nomination pools are a way to participate in staking with as little as 1 DOT and earn staking
+rewards. Nomination pools differ from custodial solutions (like staking through central exchanges)
+because they are non-custodial, native to Polkadot's protocol, permissionless, transparent, and run
+in a decentralized way by the community. Before joining a nomination pool, you must ensure that the
+pool is earning rewards and nominating the validators that match your preferences. Participating in
+pools is more of a set-and-forget action than nominating by yourself. It is worth noting that it is
+the pool operator that maintains the list of validators nominated by the pool, and so, in a way, you
+are trusting the pool operator to act in your best interests. However, it is advised to check the
+validators nominated by the pool from time to time and change the pool if necessary.
+
+|                                         Nominating                                          |                                                     Joining a Pool                                                      |
+| :-----------------------------------------------------------------------------------------: | :---------------------------------------------------------------------------------------------------------------------: |
+|                                 Minimum 10 DOT to nominate.                                 |                                              Minimum 1 DOT to be a member.                                              |
+| Annual return based on past performance is approximately 15% of the bonded funds. | Annual return based on past performance is approximately 15% of the bonded funds. |
+| Rewards can be compounded automatically or sent to any account. | Rewards must be manually claimed to compound them or send them to another account. |
```

Comment:
As far as I know, it is not possible to set a different account for rewards destination for the pool rewards, whereas for the staking rewards you can.

```suggestion
| Rewards can be compounded automatically or sent to any account. | Rewards can be manually claimed to the pool member's account and be bonded in the pool again to compound them |
```